runtime/vm/CRIUHelpers.cpp

```diff
@@ -1749,9 +1748,15 @@ criuCheckpointJVMImpl(JNIEnv *env,
 		systemReturnCode = vm->checkpointState.criuDumpFunctionPointerType();
 		VM_VMHelpers::setVMState(currentThread, J9VMSTATE_CRIU_SUPPORT_RESTORE_PHASE_START);
 		restoreNanoTimeMonotonic = j9time_nano_time();
-		restoreNanoUTCTime = j9time_current_time_nanos(&success);
-		vm->checkpointState.lastRestoreTimeMillis = (I_64)(restoreNanoUTCTime / J9TIME_NANOSECONDS_PER_MILLIS);
-		Trc_VM_criu_after_dump(currentThread, restoreNanoTimeMonotonic, restoreNanoUTCTime, vm->checkpointState.lastRestoreTimeMillis);
```

Comment:
this should not be changed either, we still need to know when execution returned to java for debugging purposes

Reply:
Keep the tracepoint as is. We need it before the first potential failure for debugging purposes. You can put the result of `(I_64)(restoreNanoUTCTime / J9TIME_NANOSECONDS_PER_MILLIS)` in a local.

Reply:
I will put the result of `(I_64)(restoreNanoUTCTime / J9TIME_NANOSECONDS_PER_MILLIS)` in a local `lastRestoreTimeMillis`, remove `lastRestoreTimeMillis` from `vm->checkpointState`, and keep `criuRestoreStartTimeInNanoseconds` in `vm->checkpointState`.

Reply:
I kept `Trc_VM_criu_after_dump` unchanged before the first potential failure and added `Trc_VM_criu_restore_start_after_dump` to display the `criuRestorePid` and `criuRestoreStartTimeInNanoseconds`. Since the original native `getLastRestoreTimeImpl` could be used for debugging purposes https://github.com/eclipse-openj9/openj9/pull/18680#discussion_r1454119453, I kept `lastRestoreTimeMillis` in `vm->checkpointState`.